### PR TITLE
Add python3 testing with tox

### PR DESCRIPTION
**Pull request by oms4suse**
This allows users to test the code with tox on python 2.7 and python 3.4

### CODE REVIEW

File: ceph_cfg/keyring.py [hunk 1]
```diff
@@ -314,9 +315,9 @@ def fset(self, name):
                 raise ValueError("Programming error for key_type with value:%s" % (name))
             try:
                 implementation.get_path_keyring()
-            except Error, e:
+            except (Error) as expt:
```
Comment (swiftgist) [inline ceph_cfg/keyring.py]: I'm bikeshedding. :)  I would've went with err.

File: ceph_cfg/rados_client.py [hunk 2]
```diff
@@ -158,7 +159,7 @@ def keyring_service_create(self):
         log.info("Make missing keyring:%s" % (self.keyring_service_path))
         keyringobj = keyring.keyring_facard(self.model)
         keyringobj.key_type = self.bootstrap_keyring_type
-        oldmask = os.umask(077)
+        oldmask = os.umask(int('077', 8))
```
Comment (swiftgist) [inline ceph_cfg/rados_client.py:162]: python 3?  Octals aren't welcomed.  Oh well.

File: ceph_cfg/util_configparser.py [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
 
 class ConfigParserCeph(ConfigParser.ConfigParser):
 
     def optionxform(self, s):
         """
         Make config files with white space use '_'
         """
-        s = string.replace(s.strip(), ' ', '_')
-        return s
+        striped = s.strip()
```
Comment (swiftgist) [inline ceph_cfg/util_configparser.py]: Did you mean stripped?

**Comment by swiftgist** [comment]
Minor comments inline

Reviewed-by: Eric Jackson ejackson@suse.com

File: ceph_cfg/util_configparser.py (hunk 3 above)
Comment (oms4suse) [inline ceph_cfg/util_configparser.py]: yep changed

File: ceph_cfg/keyring.py (hunk 1 above)
Comment (oms4suse) [inline ceph_cfg/keyring.py]: OK I changed all similar code.

File: ceph_cfg/rados_client.py (hunk 2 above)
Comment (oms4suse) [inline ceph_cfg/rados_client.py:162]: yep not welcome.

**Pull request merged**